.github/workflows/ci.yaml

```diff
@@ -37,6 +37,8 @@ jobs:
           pipx install poetry
       - name: Run tests
         run: tox run -e unit
+      - name: Upload Coverage to Codecov
+        uses: codecov/codecov-action@v3
```

Comment:
There should be a token generated by Is now.

Reply:
I think the token is necessary for v4+ only. Right?

Reply:
Yes.